etc/openqa/workers.ini

```diff
@@ -23,3 +23,8 @@
 
 #[openqa.example.host]
 # SHARE_DIRECTORY = /var/lib/openqa/example
+# In case caching of assets and tests needs to be enabled
+# CACHEDIRECTORY and TESTPOOLSERVER need to be enabled
+# CACHEDIRECTORY = /var/lib/openqa/cache
+# TESTPOOLSERVER = rsync://deimos.suse.de/openqa-tests
```

Comment:
https://github.com/coolo/openQA/blob/3a8c12e23652ee23ea798faf791c3ac91e3ba4ca/etc/openqa/workers.ini - TESTPOOLSERVER is a crappy name. If someone has a better name, shoot. But we wanted to avoid rsync in the name to have the option to extend this later to e.g. also use http.